packages/components/src/dimension-control/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import {
+	Icon,
+	SelectControl,
+} from '../';
+import { __ } from '@wordpress/i18n';
+
+import {
+	Fragment,
+} from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import sizesTable, { findSizeBySlug } from './sizes';
+
+export function DimensionControl( props ) {
```

Comment:
I'm personally a bit hesitant on making this a public component in `@wordpress/components`.

The `@wordpress/components` package is basically the design system while this is more something for the editor, maybe something for the `block-editor` package?

What do you all think about moving this to the block-editor package or making it experimental?

Reply:
Nvm, I just saw that it's already experimental :) I wonder if we should move it to the block-editor and stabilize it once we use it a block.